src/azure-cli/azure/cli/command_modules/appservice/_help.py

```diff
@@ -1542,3 +1542,167 @@
     type: command
     short-summary: Creates a remote connection using a tcp tunnel to your web app
 """
+
+helps['functionapp hybrid-connection'] = """
+    type: group
+    short-summary: commands that list, add, or remove hybrid connections from a sepcific functionapp
+"""
+
+helps['functionapp hybrid-connection list'] = """
+    type: command
+    short-summary: lists all the hybrid connections set on the specified functionapp
+    examples:
+        - name: list the hybrid connections
+          text: >
+              az functionapp hybrid-connection list -g [myResourceGroup] -n [myAppName] -s [slotName]
```

Comment:
I don't think you need to wrap `[` around a value which already served as a sample one like `myResourceGroup`